src/site/_includes/components/updates/updates.njk

```diff
@@ -5,6 +5,8 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds-from-desktop">
           <h2 class="govuk-heading-l">Updates</h2>
+          <p class="govuk-body"><span class="govuk-!-font-weight-bold">20 October 2023:</span> The Defence Developer Services (D2S) will shortly commence work on its re-use strategy and the discovery and delivery of a re-use components service. As part of this, we will commence a Discovery activity for the Design System Toolkit. During the Discovery Phase, the MOD.UK Frontend and Design Library will be in maintenance mode with no further enhancements being made during this time. Only high priority issues such as security fixes or outages will be completed.</p>
+          <p class="govuk-body"><u><span class="govuk-!-font-weight-bold govuk-!-font-underlined">Getting support</span></u></br>For high priority issues, you can contact <a href="mailto:design-system@digital.mod.uk">design-system@digital.mod.uk</a>. If you have any general questions, you can contact <a href="mailto:UKStratComDD-Foundry-DSO-Engage@mod.gov.uk">UKStratComDD-Foundry-DSO-Engage@mod.gov.uk</a>.</p>
           <p class="govuk-body"><span class="govuk-!-font-weight-bold">2 May 2023:</span> We’ve updated MOD.UK Frontend to include improvements from <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.6.0" class="govuk-link moduk-link--maroon">GOV.UK Frontend v4.6.0</a>. This includes visual improvements to some components.</p>
```

Comment:
I think this should be removed, so there's only the latest update displayed.

Reply:
I asked Becca about this and she said to keep it in